packages/stops-overlay/src/index.js

```diff
@@ -28,8 +29,20 @@ class StopsOverlay extends MapLayer {
   }
 
   refreshStops = () => {
-    const { leaflet, minZoom, refreshStops } = this.props;
-    if (leaflet.map.getZoom() < minZoom) {
+    const { leaflet, refreshStops, symbols } = this.props;
+
+    // Force the map to render no symbols if the map zoom level
+    // is less than the farthest zoom level at which symbols are defined.
+    const minZoomForUpdate = symbols
+      ? symbols.reduce((lowestZoom, level) => {
+          if (level.minZoom < lowestZoom) {
+            return level;
```

Comment:
Return `level`? Isn't that an object?  I think this function could be rewritten as:

```js
(lowestZoom, level) => Math.min(level.minZoom, lowestZoom)
```

Reply:
Good call!

Reply:
Addressed in https://github.com/opentripplanner/otp-ui/pull/159/commits/6de060a44452342c50e55f3e017bcc1e02aa6f19.